src/UuidType.php

```diff
@@ -80,6 +80,10 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform)
         if (empty($value)) {
             return null;
         }
+        
+        if ($value instanceof UuidInterface) {
+            return $value->toString();
+        }
```

Comment:
Since `Ramsey\Uuid\Uuid` implements `UuidInterface`, you can get rid of these lines and change the `if` statement below to `UuidInterface` from `Uuid`.

Reply:
don't static analysis will raise an error as `UuidInterface` doesn't have the method `__toString` so any other implementation of the interface than `Uuid` may not have the method, resulting in an exception here ?